application/controllers/util.js

```diff
@@ -31,10 +31,24 @@ module.exports = {
       return userCtrl.find({
         $or: [
           {
-            username: username
+            $and: [
+              {
+                username: new RegExp(username, 'i')
```

Comment:
Look for both statements if they can be reduced by the same method as in handler.js